subsys/mpsl/mpsl_init.c

```diff
@@ -22,10 +22,14 @@ LOG_MODULE_REGISTER(mpsl_init, CONFIG_MPSL_LOG_LEVEL);
 #endif
 #define MPSL_LOW_PRIO (4)
 
+#define MPSL_SCHEDULER_INIT CONFIG_MPSL_SCHEDULER_INIT
```

Comment:
Do we need a define that maps directly to the define created by Kconfig?

Reply:
I think I can remove it